rasa/server.py

```diff
@@ -377,6 +379,28 @@ def validate_request_body(request: Request, error_message: Text) -> None:
         raise ErrorResponse(HTTPStatus.BAD_REQUEST, "BadRequest", error_message)
 
 
+def validate_events_in_request_body(request: Request) -> None:
+    """Validates events format in request body."""
+    from jsonschema import validate
+    from jsonschema import ValidationError
+
+    if not isinstance(request.json, list):
+        events = [request.json]
+    else:
+        events = request.json
+
+    try:
+        validate(events, EVENTS_SCHEMA)
+    except ValidationError as error:
+        raise ErrorResponse(
+            HTTPStatus.BAD_REQUEST,
+            "BadRequest",
+            f"Failed to validate the events format. "
+            f"For more information about the format visit the docs. Error: {error}",
+            help_url=DOCS_BASE_URL_ACTION_SERVER + "/pages/action-server-api",
```

Comment:
* let's use a f-string here please
* shouldn't we rather redirect them to the API spec for the Rasa Open Source server endpoints? That's [here](https://rasa.com/docs/rasa/pages/http-api)

Reply:
Yup, will change to f-string. I didn't know of this page, thanks for pointing this out - would the specific section be [this](https://rasa.com/docs/rasa/pages/http-api#tag/Tracker)? particularly the POST `append events to a tracker` and PUT `replace a tracker's events`?

Reply:
Exactly 👍🏻 Probably good enough to just link to the overall page instead of the individual sections